### ISSUE TITLE
Share profile on social media

### ISSUE THREAD

**Issue by jahlela** [Ledgy/TrainTrack#11]
**Description**
* Create design proposal for adding social media sharing buttons for the profile page (include screenshots)
* Add social sharing for Facebook and twitter (MVP)

**Comment by Vamshi0104** [comment]
<!DOCTYPE html>
<html>
<title>Social Media</title>
<head>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.0/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">

</head>
<body>

<style>
body
{
background-color:black;
}
.svg--source {
  display: none;
}
.svg--icon {
  width: 100%;
  max-width: 5rem;
  height: 100%;
  max-height: 5rem;
  display: block;
  margin: 0 auto;
  fill: currentColor;
}
.wrapper {
  width: 100%;
  height: 100vh;
  display: flex;
  flex-wrap: wrap;
  align-items: center;
  justify-content: center;
  padding: 1rem;
}
.share {
  width: 5rem;
  height: 5rem;
  float: left;
  margin: .5rem 1rem .5rem 0;
  color: #353c4a;
  border: .125rem solid #f3f3f3;
  box-shadow: 0 0 8px 0 rgba(50, 50, 50, 0.15);
  border-radius: 50%;
  transition: 250ms;
}
.share:last-child {
  margin-right: 0;
}
.share:focus {
  outline-color: inherit;
}
.twitter:hover, .twitter:focus {
  color: #00ACED;
  box-shadow: 0 0 24px 0 #00ACED;
}
.instagram:hover, .instagram:focus {
  color: #8a3ab9;
  box-shadow: 0 0 24px 0  #8a3ab9;
}
.github:hover, .github:focus {
  color: green;
  box-shadow: 0 0 24px 0 green;
}
.facebook:hover, .facebook:focus {
  color: #3b5998;
  box-shadow: 0 0 24px 0 #3b5998;
}
h3
{
font-family:cursive;
color:lime;
}
</style>

<svg class="svg--source" width="0" height="0" aria-hidden="true">
 <symbol id="svg--twitter" viewbox="0 -7 15 30">
    <path d="M15.36 3.434c-0.542 0.241-1.124 0.402-1.735 0.476 0.624-0.374 1.103-0.966 1.328-1.67-0.583 0.346-1.23 0.598-1.917 0.733-0.551-0.587-1.336-0.954-2.205-0.954-1.668 0-3.020 1.352-3.020 3.019 0 0.237 0.026 0.467 0.078 0.688-2.51-0.126-4.735-1.328-6.224-3.155-0.261 0.446-0.41 0.965-0.41 1.518 0 1.048 0.534 1.972 1.344 2.514-0.495-0.016-0.961-0.151-1.368-0.378 0 0.013 0 0.025 0 0.038 0 1.463 1.042 2.683 2.422 2.962-0.253 0.069-0.52 0.106-0.796 0.106-0.194 0-0.383-0.018-0.568-0.054 0.384 1.2 1.5 2.073 2.821 2.097-1.034 0.81-2.335 1.293-3.75 1.293-0.244 0-0.484-0.014-0.72-0.042 1.336 0.857 2.923 1.357 4.63 1.357 5.554 0 8.592-4.602 8.592-8.593 0-0.13-0.002-0.261-0.009-0.39 0.59-0.426 1.102-0.958 1.507-1.563z"/>
  </symbol>
<symbol id="svg--instagram" viewbox="0 -7 23 35">
<path d="M7.8,2H16.2C19.4,2 22,4.6 22,7.8V16.2A5.8,5.8 0 0,1 16.2,22H7.8C4.6,22 2,19.4 2,16.2V7.8A5.8,5.8 0 0,1 7.8,2M7.6,4A3.6,3.6 0 0,0 4,7.6V16.4C4,18.39 5.61,20 7.6,20H16.4A3.6,3.6 0 0,0 20,16.4V7.6C20,5.61 18.39,4 16.4,4H7.6M17.25,5.5A1.25,1.25 0 0,1 18.5,6.75A1.25,1.25 0 0,1 17.25,8A1.25,1.25 0 0,1 16,6.75A1.25,1.25 0 0,1 17.25,5.5M12,7A5,5 0 0,1 17,12A5,5 0 0,1 12,17A5,5 0 0,1 7,12A5,5 0 0,1 12,7M12,9A3,3 0 0,0 9,12A3,3 0 0,0 12,15A3,3 0 0,0 15,12A3,3 0 0,0 12,9Z" />
  </symbol>
<symbol id="svg--facebook" viewbox="0 -7 16 30">
    <path d="M12 3.303h-2.285c-0.27 0-0.572 0.355-0.572 0.831v1.65h2.857v2.352h-2.857v7.064h-2.698v-7.063h-2.446v-2.353h2.446v-1.384c0-1.985 1.378-3.6 3.269-3.6h2.286v2.503z" />
  </symbol>
<symbol id="svg--github" viewbox="-30 -30 150 150">
    <path d="M61.896,52.548c-3.59,0-6.502,4.026-6.502,8.996c0,4.971,2.912,8.999,6.502,8.999
        c3.588,0,6.498-4.028,6.498-8.999C68.395,56.574,65.484,52.548,61.896,52.548z M84.527,29.132c0.74-1.826,0.777-12.201-3.17-22.132
        c0,0-9.057,0.993-22.76,10.396c-2.872-0.793-7.736-1.19-12.597-1.19s-9.723,0.396-12.598,1.189C19.699,7.993,10.645,7,10.645,7
        c-3.948,9.931-3.913,20.306-3.172,22.132C2.834,34.169,0,40.218,0,48.483c0,35.932,29.809,36.508,37.334,36.508
        c1.703,0,5.088,0.004,8.666,0.009c3.578-0.005,6.965-0.009,8.666-0.009C62.191,84.991,92,84.415,92,48.483
        C92,40.218,89.166,34.169,84.527,29.132z M46.141,80.574H45.86c-18.859,0-33.545-2.252-33.545-20.58
        c0-4.389,1.549-8.465,5.229-11.847c6.141-5.636,16.527-2.651,28.316-2.651c0.045,0,0.093-0.001,0.141-0.003
        c0.049,0.002,0.096,0.003,0.141,0.003c11.789,0,22.178-2.984,28.316,2.651c3.68,3.382,5.229,7.458,5.229,11.847
        C79.686,78.322,65,80.574,46.141,80.574z M30.104,52.548c-3.588,0-6.498,4.026-6.498,8.996c0,4.971,2.91,8.999,6.498,8.999
        c3.592,0,6.502-4.028,6.502-8.999C36.605,56.574,33.695,52.548,30.104,52.548z" />
  </symbol>
</svg>
<div class="wrapper">
<div class="connect">
<a href="" class="share github" title="Git Hub">
    <svg role="presentation" class="svg--icon">
      <use xlink:href="#svg--github" />
      <span class="clip"></span>
    </svg>
  </a>
<a href="" class="share twitter" title="Twitter">
    <svg role="presentation" class="svg--icon">
      <use xlink:href="#svg--twitter" />
    </svg>
    <span class="clip"></span>
  </a>
<a href="example.html" class="share instagram" title="Instagram">
    <svg role="presentation" class="svg--icon">
      <use xlink:href="#svg--instagram" />
    </svg>
    <span class="clip"></span>
  </a>
<a href="" rel="author" class="share facebook" title="Face Book">
    <svg role="presentation" class="svg--icon">
      <use xlink:href="#svg--facebook" />
      <span class="clip"></span>
    </svg>
  </a>
<br><br><br><br>
<h3 align="center">Happy Coding .... :) </h3>
</div>

</body>
</html>

Below are attached ScreenShots of Output Generated when above .html file is saved and run.


![1](https://user-images.githubusercontent.com/39955657/67225939-b9e79100-f451-11e9-89c9-769458d7aa1a.png)
![2](https://user-images.githubusercontent.com/39955657/67225941-ba802780-f451-11e9-8419-d4665b7d0633.png)
![3](https://user-images.githubusercontent.com/39955657/67226932-c371f880-f453-11e9-9ad5-e1bf4ff1922d.png)
 
Hope u Liked it........
